crates/ruff_linter/src/registry/rule_set.rs

```diff
@@ -291,7 +291,9 @@ impl Display for RuleSet {
         } else {
             writeln!(f, "[")?;
             for rule in self {
-                writeln!(f, "\t{rule:?},")?;
+                let name = rule.as_ref();
```

Comment:
Should we change this to use kebab-case while we’re here?

Reply:
That's what `.as_ref()` does :)